internal/rule/flatcase_test.go

```diff
@@ -0,0 +1,98 @@
+package rule
+
+import (
+	"testing"
+)
+
+func TestFlatcase(t *testing.T) {
+	rule := new(Flatcase).Init()
+
+	validTests := map[string]string{
+		"expected use case":                  "abc0123",
+		"lowercase letters":                  "abcdefghijklmnopqrstuvwxyz",
+		"lowercase letters with digits":      "0123456789",
+		"one lowercase letter":               "a",
+		"one digit":                          "1",
+		"lower case letters with diacritics": "åäöéèêçñ",
+		"lower case cyrillic letters":        "абвгдежзийклмнопрстуфхцчшщъыьэюя",
+		"lower case greek letters":           "αβγδεζηθικλμνξοπρστυφχψω", // Greek alpha lower case letter
+		"lower case armenian letters":        "աաբգդեէզէթժիյկլհմնոօփքռստուֆք",
+	}
+
+	for name, value := range validTests {
```

Comment:
For records, I didn't reuse the `ruleTest` struct available in [rule_test.go](https://github.com/loeffel-io/ls-lint/blob/v2.3.0/internal/rule/rule_test.go) because I wanted to write tests differently.

Reply:
Please use the ruleTest struct to keep consistency over all tests

Reply:
Can I add a name string in ruleTest struct then ?